.cirrus.yml

```diff
@@ -38,4 +38,4 @@ windows_task:
     - powershell hack/configure-windows-ci.ps1
     - refreshenv
     - go install .\cmd\nerdctl\
-    - go test -v ./cmd/...
+    - go test -v ./integration/...
```

Comment:
The "Test suite" section in `README.md` has to be updated too
https://github.com/containerd/nerdctl/tree/v1.1.0#test-suite